src/main/java/com/reto/usuario/application/handler/implementation/UserServiceImpl.java

```diff
@@ -21,8 +22,9 @@ public class UserServiceImpl implements IUserService {
     private final IUserResponseMapper userResponseMapper;
 
     @Override
-    public void registerUserWithOwnerRole(UserRequestDto userRequestDto) {
-        userUseCasePort.registerUserWithOwnerRole(userRequestMapper.toUserModel(userRequestDto));
+    public UserOwnerResponseDto registerUserWithOwnerRole(UserRequestDto userRequestDto) {
+        return userResponseMapper.toUserOwnerResponseDto(userUseCasePort.registerUserWithOwnerRole(
```

Comment:
priorizar el código legible, por ejemplo:
public UserOwnerResponseDto registerUserWithOwnerRole(UserRequestDto userRequestDto) {
    final UserModel userRequestModel = this.userRequestMapper.toUserModel(userRequestDto);
    final UserModel userRegisteredModel = this.userUseCasePort.registerUserWithOwnerRole(userRequestModel );
    return this.userResponseMapper.toUserOwnerResponseDto(userRegistered );
}

Reply:
Cambios agregados